packages/hyprls/package.yaml

```
@@ -0,0 +1,16 @@
+---
+name: hyprls
+description: A LSP server for Hyprland config files
+homepage: https://github.com/hyprland-community/hyprls
+licenses:
+  - MIT
+languages:
+  - Hypr
+categories:
+  - LSP
+
+source:
+  id: pkg:golang/github.com/ewen-lbh/hyprls/cmd/hyprls@v0.2.0
```

While this purl works it's not correctly identifying the package, we can fix this by specifying the submodule in the subpath 
```suggestion
  id: pkg:golang/github.com/ewen-lbh/hyprls@v0.2.0#cmd/hyprls
```